cmake/Packing.cmake

```diff
@@ -57,12 +58,10 @@ set(CPACK_PACKAGE_CONTACT "CARMAsupport@dot.gov")
 set(CPACK_DEBIAN_PACKAGE_MAINTAINER "FHWA Saxton Laboratory <${CPACK_PACKAGE_CONTACT}>")
 
 # license file not currently used in Debian packaging
-#set(CPACK_RESOURCE_FILE_LICENSE "$ENV{CARMA_OPT_DIR}/LICENSE")
+set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_CURRENT_SOURCE_DIR}/LICENSE")
```

Comment:
Does this fix the issue with the license not showing up? Can we remove the comment above this line?